### ISSUE TITLE
TCK: revisit short delays handling

### ISSUE THREAD

**Issue by xstefank** [eclipse/microprofile-lra#250]
There are some concerns mentioned around short delays in TCK in https://github.com/eclipse/microprofile-lra/pull/240 discussions. Proposed options:

- replace short delay calls with calls to LraRecoveryService
- replace short delay calls with something like:

```java
boolean waitForCallbacks(); // returns after an attempt to notify all participants was made
boolean retryCallbacks(); // some callbacks previously failed so request redelivery
```

**Comment by mmusgrov** [comment]
> 
> * replace short delay calls with calls to LraRecoveryService
> * replace short delay calls with something like:

waitForCallbacks would be the equivalent of the short delay
retryCallbacks would be the equivalent of the long delay.

The names could be improved but that is the intent.

> 
> ```java
> boolean waitForCallbacks(); // returns after an attempt to notify all participants was made
> boolean retryCallbacks(); // some callbacks previously failed so request redelivery
> ```

**Comment by mmusgrov** [comment]
Can we close this one since it covers the same problem area as #249 and the commentary is getting hard to follow since the discussion is now spanning both issues and it is getting confusing as to what has/has not already been said.

**Comment by xstefank** [comment]
ok, I will do in one PR.
